src/content/Blog/running-vllm-on-akash/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Running vLLM on Akash"
+description: Learn how to save money by replacing Closed AI models like GPT-4 by running open source LLM's on Akash with vLLM.  vLLm is the most powerful open source LLM server for multiuser / high concurrent request LLM services.  We show you how to run fully featured open source llm application stack on Akash including with OpenWeb UI Chat application and AI Agent workflows for using LangChain and Crew AI.    
+
+pubDate: "2024-5-8"
```

Comment:
Will need to update this before publishing

Reply:
@zachhorn i have updated the pubDate field to today.  Do you want a different one?